### PR TITLE
v0.2.0

### PR DESCRIPTION
**Pull request by mikheevm**
**Changes:**
- Improve type definitions

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,27 +20,46 @@ export interface SafeInfo {
...
+
+export interface ToMessageToPayload {
+  [TO_SAFE_MESSAGES.SEND_TRANSACTIONS]: Transaction[];
 }
 
-export enum ToSafeMessages {
-  SEND_TRANSACTIONS = 'SEND_TRANSACTIONS',
+export type ToSafeMessages = typeof TO_SAFE_MESSAGES;
+
+const FROM_SAFE_MESSAGES = {
```
Comment (nicosampler) [inline src/index.ts:43]: could it be an enum instead?

File: src/index.ts [hunk 2]
```diff
@@ -13,27 +20,46 @@ export interface SafeInfo {
...
-  ON_SAFE_INFO = 'ON_SAFE_INFO',
+interface CustomMessageEvent extends MessageEvent {
+  data: {
+    messageId: keyof FromSafeMessages;
+    data: FromMessageToPayload[keyof FromSafeMessages];
+  };
+}
+
+export const TO_SAFE_MESSAGES = {
+  SEND_TRANSACTIONS: 'SEND_TRANSACTIONS' as const,
```
Comment (nicosampler) [inline src/index.ts:34]: same here

Comment (mikheevm) [reply src/index.ts:34]: I couldn't make this interface work with an enum, this is why I changed it:
https://github.com/gnosis/safe-apps-sdk/pull/25/files#diff-f41e9d04a45c83f3b6f6e630f10117feR37

Comment (mikheevm) [reply src/index.ts:34]: If you can make it work I have nothing against it, but I also don't see any reasons to switch to enum

Comment (mikheevm) [reply src/index.ts:34]: https://stackoverflow.com/questions/39701524/using-enum-as-interface-key-in-typescript

**Pull request merged**